### PR TITLE
fix unstarted core-ci

### PR DESCRIPTION
**Pull request by Netyyyy**

### CODE REVIEW

**Review by chenrujun — APPROVED** [review]
LGTM.

File: sdk/spring/ci.yml [hunk 1]
```diff
@@ -330,17 +330,17 @@ extends:
...
+              SpringCloudVersion: 2021.0.1
+      - stage: TestCompatibleVersion
         jobs:
           - template: ../../../../sdk/spring/spring-sdk-client-test.yml
             parameters:
               Artifacts: ${{ parameters.Artifacts }}
-              SpringBootVersion: 2.5.7
-              SpringCloudVersion: 2020.0.4
+              SpringBootVersion: 2.6.2
+              SpringCloudVersion: 2021.0.0
```
Comment (saragluna) [inline sdk/spring/ci.yml:346]: does this mean we will not test the compatibility with Spring Boot 2.5.x?

Comment (chenrujun) [reply sdk/spring/ci.yml:346]: It should be Spring Boot 2.5.x.
@Netyyyy could you please fix it?

1. Use the latest patch version of Spring Boot 2.5.x. Now it's 2.5.10. 
2. Find latest compatible spring cloud version. Now it's 2020.0.5. 

Refs: 
1. https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/
2. https://spring.io/projects/spring-cloud 
3. https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/

Comment (Netyyyy) [reply sdk/spring/ci.yml:346]: @chenrujun fixed in #27535, will change to 2.5.10 on next version update

**Pull request merged**